### ISSUE TITLE
PokeClicker update v.0.10.14. broke some scripts

### ISSUE THREAD

**Issue by K0STYURIK** [Ephenia/Pokeclicker-Scripts#342]
SyntheticShinySynapse and IniniteSeasonalEvents both not appear in the menu at all. Not functioning not on desktop nor on browser

EnhancedAutoClicker works fine with wild Pokemon, but AutoGYM and AutoDungeon only work in browser version, on desktop version autoclicking in GYMs and dungeons is stuck (timer is stuck too for some reason).

Not sure if is a right place to mention it, but buttons from AutoSafariZone script by Kanzen01 not appearing in browser at all. Activating AutoSafariZone on desktop doesn't do anything.

**Comment by tisserm** [comment]
I also have the timer issue, but on browser version so I don't know if the problem is the same (I use Firefox btw). It is stuck in dungeons, battle frontier, flutes, items and so on -> it completely prevents battle frontier from working in my case.

Not OP but i have the following error for IniniteSeasonalEvents: 
> TypeError: SpecialEvents.newEvent is not a function
initEvents moz-extension://57920583-cd2e-443e-b92f-58215958bb9f/userscripts/[Pokeclicker]-Infinite-Seasonal-Events.user.js?id=78c27b44-8dd5-4834-ba41-6236650bc9ac:31
    hideSplashScreen moz-extension://57920583-cd2e-443e-b92f-58215958bb9f/userscripts/[Pokeclicker]-Infinite-Seasonal-Events.user.js?id=78c27b44-8dd5-4834-ba41-6236650bc9ac:238
    hideSplashScreen moz-extension://57920583-cd2e-443e-b92f-58215958bb9f/userscripts/[Pokeclicker]-Oak-Items-Unlimited.user.js?id=5a4fa68f-f6ee-4b7c-b821-eaaac7439f29:36
    hideSplashScreen moz-extension://57920583-cd2e-443e-b92f-58215958bb9f/userscripts/[Pokeclicker]-Overnight-Berries.user.js?id=8e53f5fb-9f09-42da-9949-18af5cc0e4f3:214
    hideSplashScreen moz-extension://57920583-cd2e-443e-b92f-58215958bb9f/userscripts/[Pokeclicker]-Perky-Pokerus-Pandemic.user.js?id=fb8261eb-ee70-4309-a943-205522877a37:49
    hideSplashScreen moz-extension://57920583-cd2e-443e-b92f-58215958bb9f/userscripts/[Pokeclicker]-Simple-Weather-Changer.user.js?id=f7b089f1-b85d-4449-8509-f5857765df9d:67
    start https://www.pokeclicker.com/scripts/script.min.js?v=0.10.14:37
    promise callback*start https://www.pokeclicker.com/scripts/script.min.js?v=0.10.14:19
    getTrainerCard webpack://pokeclicker/./src/modules/profile/Profile.ts?:43
    getTrainerCard webpack://pokeclicker/./src/modules/profile/Profile.ts?:35
    getTrainerCard webpack://pokeclicker/./src/modules/SaveSelector.ts?:70
    loadSaves webpack://pokeclicker/./src/modules/SaveSelector.ts?:24
    loadSaves webpack://pokeclicker/./src/modules/SaveSelector.ts?:23
    \<anonymous> https://www.pokeclicker.com/scripts/script.min.js?v=0.10.14:5553
    EventListener.handleEvent* https://www.pokeclicker.com/scripts/script.min.js?v=0.10.14:5541

**Comment by K0STYURIK** [comment]
> I haven’t had any issues with Auto Gym or Auto Dungeon on desktop. What specifically are you trying to use them for? Can you provide the console log?

Now EnhancedAutoClicker on desktop doesn't work for me at all. It is enabled in settings
![Image 001](https://github.com/Ephenia/Pokeclicker-Scripts/assets/115414677/c202969b-60a9-484c-aedb-27844ce788c9)
But there are nothing on the wild Pokemon screen (there was usual AutoClicker UI when I opened game the last time)
![Image 002](https://github.com/Ephenia/Pokeclicker-Scripts/assets/115414677/04072806-ba3f-4ae9-9e24-d20e873b34a1)
Nothing in dungeons
![Image 003](https://github.com/Ephenia/Pokeclicker-Scripts/assets/115414677/60a574a3-113a-4d98-8db0-727b74fdcdbb)
Nothing in gyms
![Image 004](https://github.com/Ephenia/Pokeclicker-Scripts/assets/115414677/7951fd30-c0be-4bf0-8b1e-83c3beebcbdc)
For some reason timer is still. 

Working totally fine on browser. 

I'd be glad to provide the console log if you teach me how to do it

**Comment by K0STYURIK** [comment]
> Bug reports for the Auto Safari Zone script are better off in the PR as it’s not yet an official script.

Me and other person (Suitov) already commented in AutoSafariZone PR, I just mentioned it here in case it's somehow related, and the problem with other three scripts causing the problem with AutoSafariZone or vise versa

**Comment by Player24826** [comment]
My farm has froze since the update.

**Comment by Player24826** [comment]
Keep in mind I know nothing of code but could the time related code have been moved to the  app.asar file?

**Comment by Player24826** [comment]
![Screenshot 2023-08-21 005108](https://github.com/Ephenia/Pokeclicker-Scripts/assets/142757705/ae9e56a0-0d57-42a2-b38b-f3b27b91692c)
![Screenshot 2023-08-21 005218](https://github.com/Ephenia/Pokeclicker-Scripts/assets/142757705/286e1046-08e9-4b35-b21f-7ce07459232d)
![Screenshot 2023-08-21 005406](https://github.com/Ephenia/Pokeclicker-Scripts/assets/142757705/9ee46090-dc97-4f6a-b731-f8d8f6cdf9ea)

**Comment by Player24826** [comment]
It was Infinite Seasonal Events script that broke.

**Comment by JohnyDL** [comment]
So here I have all the scripts running
![image](https://github.com/Ephenia/Pokeclicker-Scripts/assets/5202568/a4adc08b-7fe2-4cae-898f-3d030e0f603e)

? Additional Visual Settings -> Nothing looks different probably works
? Auto Battle Frontier -> Not doing the battle frontier yet in this run
Y Auto Battle Items
N Auto Quest Completer -> I had a full list of quests but back to vanilla questing
N Auto Safari Zone -> This is what I was focusing on now not there
Y Catch Filter Fantasia
Y Catch Speed Adjuster
Y Challenge Mode Changer
Y Discord Code Generator
Y Enhanced Auto Clicker
Y Enhanced Auto Hatchery
Y Enhanced Auto Mine
? Infinite Seasonal Events -> Not sure where this is located
? Oak Items Unlimited -> have all items don't want to remove any incase I can't put them back
? Omega Protein Gains -> not enough vitamins to test
? Overnight Berries -> not got any offline time to test with
Y Perky Pokerus Pandemic
Y Simple Auto Farmer
N Simple Weather Changer -> no dropdown to change weather

I have these errors in console
![image](https://github.com/Ephenia/Pokeclicker-Scripts/assets/5202568/2ba4ce59-1a6f-440c-96e2-291c1dc4cc72)
![image](https://github.com/Ephenia/Pokeclicker-Scripts/assets/5202568/01195936-a4bb-4101-837f-bf41c30c31de)
![image](https://github.com/Ephenia/Pokeclicker-Scripts/assets/5202568/d854f71c-0585-4ac3-af2a-8bc8dd124be0)

[Player24826](https://github.com/Player24826) suggested infinite seasonal events was the issue so I turned it off and everything else appears to have started to work again so I can tentatively confirm that is probably the script with an issue

**Comment by MrShiluk33** [comment]
OmegaProtein dont work yep. So if you were over max protein before and try yo add more it goes back to max possible.

**Comment by K0STYURIK** [comment]
Disabling InfiniteSeasonalEvents fixes the problems with Autoclicker and SyntheticShinySynapse

**Comment by MrShiluk33** [comment]
@K0STYURIK do you think disabling InfiniteSeasonalEvents also fixes OmegaProtein ?

**Comment by K0STYURIK** [comment]
> @K0STYURIK do you think disabling InfiniteSeasonalEvents also fixes OmegaProtein ?

Sorry, but I don't know. I never used OmegaProteinGains, so you probably should check it by yourself. Please let us know of the results anyway.

**Comment by MrShiluk33** [comment]
Yeah tested it and it works without InfiniteSeasonalEvents ! @K0STYURIK

**Comment by K0STYURIK** [comment]
> Yeah tested it and it works without InfiniteSeasonalEvents ! @K0STYURIK

Congratulations! Well, now we know InfiniteSeasonalEvents is the big bad boy breaking every other script.

**Comment by MrShiluk33** [comment]
![image](https://github.com/Ephenia/Pokeclicker-Scripts/assets/96082745/e608100d-b48e-4e21-9d77-264549ad329b)
@umbralOptimatum Pikachu (clone) isn't visible on "Mewtwo strikes back!" possible mobs available in InfiniteSeasonalEvents
